schema/deploy/shipping/views.sql

```diff
@@ -728,4 +728,53 @@ comment on view shipping.return_results_v2 is
     'Version 2 of view of barcodes and presence/absence results for return of results on website';
 
 
+create or replace view shipping.hcov19_observation_with_presence_absence_result_v1 as
+
+select
+        presence_absence.created::date as sample_tested_date,
+        present as test_result,
+        sample.details->>'sample_origin' as sample_manifest_origin,
+        best_available_encounter_date,
+        age_range_fine,
+        sex,
+        residence_puma,
+        best_available_site,
+        individual,
+        sample.identifier as sample,
+        symptoms,
+        best_available_site_type
+
+    from shipping.sample_with_best_available_encounter_data_v1
+    join warehouse.sample using (sample_id)
+    full outer join shipping.incidence_model_observation_v3 on
+      (sample.identifier = incidence_model_observation_v3.sample)
+    left join warehouse.presence_absence using (sample_id)
+    left join warehouse.target using (target_id)
+    left join warehouse.organism using (organism_id)
+    -- TODO we'll replace this via a replacement join on
+    -- incidence_model_observation_v4 with PUMAs once it's merged to master
+    left join (
+      select
+        encounter_id,
+        tract.hierarchy->'puma' as residence_puma,
+        tract.hierarchy->'neighborhood_district' as residence_neighborhood_district
+      from warehouse.encounter_location
+      left join warehouse.location as address using (location_id)
+      left join warehouse.tract as tract on (address.hierarchy -> 'tract') = tract.identifier
+      where relation = 'residence'
+      or relation = 'lodging'
+    ) as residence using (encounter_id)
+    where (
+        lineage ~ 'Human_coronavirus.2019'
+        or best_available_encounter_date >= '2020-03-01'::date
```

Comment:
> Hmm, won't this return a row for every non-hCoV-19 presence absence result too? I might be missing something, but I think this filter isn't doing what we want.

Oh, derp. I'll fix that.

> The `~ [lquery]` comparison is equivalent to `= [ltree]` when the `lquery` contains no wildcards, but the latter seems more clear. I had to go lookup what `~` was doing here, but everyone knows what `=` means.

Thanks for the info. I found the `~` operator on the [table in these Postgres docs](https://www.postgresql.org/docs/10/ltree.html) but I missed the paragraph at the top saying `=` was also available. 

> It would also be good to document with a comment why this magic date value is what it is.

Yes, good point. As far as I know it is what it is because Mike said so. But I'll investigate and add a comment. :smile: